tests/test-backend-ops.cpp

```diff
@@ -3615,6 +3616,13 @@ struct test_mul_mat_id : public test_case {
             out = ggml_add(ctx, out, out2);
         }
 
+        if (mul) {
+            std::array<int64_t, 4> ne { 1, out->ne[1], out->ne[2], out->ne[3] };
+            ne[0] = 1;
+            ggml_tensor * m = ggml_new_tensor(ctx, out->type, 4, ne.data());
+            out = ggml_mul(ctx, out, m);
+        }
```

Comment:
I really don't like how we keep making the basic, single op test cases more complex by adding flags to change the behavior completely. Can you move the fusion cases to a different class?

Reply:
sure, done.